internal/core/src/segcore/SegmentSealedImpl.cpp

```diff
@@ -893,7 +897,10 @@ std::tuple<std::string, std::shared_ptr<ColumnBase>> static ReadFromChunkCache(
     const storage::ChunkCachePtr& cc,
     const std::string& data_path,
     const storage::MmapChunkDescriptorPtr& descriptor) {
-    auto column = cc->Read(data_path, descriptor);
+    // For mmap mode, field_meta is unused, so just construct a fake field meta.
+    auto fm = FieldMeta(FieldName(""), FieldId(0), milvus::DataType::NONE);
```

Comment:
Is there a way to specify the right data type even though it is not need for now ?